charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -29,6 +29,9 @@ metadata:
   {{- end }}
 rules:
   # Read
+  - apiGroups: ["coordination.k8s.io"]
```

Comment:
We also don't need clusterrole on this. We just need permission from the single namespace. Is it possible to only give ourselves permission to the single namespace where we need the permission?